rpmlb/cli.py

```diff
@@ -17,6 +17,10 @@
 @click.option(
     '--verbose', '-v', is_flag=True, default=False,
     help='Turn on verbose logging.',
+    # Enable logging as early as possible
+    is_eager=True, expose_value=False,
+    callback=lambda ctx, param, verbose:
+        rpmlb_logging.configure_logging(verbose),
```

Comment:
This `configure_logging` is called by both cases of with `--verbose` option and without `--verbose` option, right?